backend/package.json

```diff
@@ -14,6 +14,7 @@
     "lint:type": "npx tsc --noEmit",
     "lint:js": "npx eslint \"src/**/*.ts\"",
     "start:dev": "nodemon --exec \"node --loader ts-paths-esm-loader\" src/index.ts",
+    "seed:dev": "node --loader ts-paths-esm-loader ../node_modules/knex/bin/cli.js seed:run",
```

Comment:
we also need to add seed command for production and and it to restart command, take a look at the migrate command